tests/reftests/reinstall.test

```diff
@@ -85,3 +85,88 @@ The following actions would be performed:
   - recompile b 1
 === install 1 package
   - install   c 1
+### :::::: Check that reinstalling a non-installed package is equivalent to installing it
+### :::::: TODO: This is currently broken
+### <pkg:d.1>
+opam-version: "2.0"
+### opam reinstall d --cudf=reinstall --show --yes
+d is not installed. Install it? [y/n] y
+The following actions would be performed:
+=== install 1 package
+  - install d 1
+### opam install d --cudf=install --show --yes
+The following actions would be performed:
+=== install 1 package
+  - install d 1
+### diff -u install-1.cudf reinstall-1.cudf
```

Comment:
```suggestion
### diff -U 0 install-1.cudf reinstall-1.cudf
```
if portable ?

Reply:
I used `diff` without argument instead. That should be the most portable